generators/base/index.js

```diff
@@ -271,11 +271,17 @@ module.exports = class extends Generator {
   }
 
   install() {
+    const reactNavigavtion = [
```

Comment:
`Navigavtion`?

Reply:
Good spot 😅

Reply:
I kinda like it, you could change it to `Navigav` 😆